### DISCUSSION TITLE
Learning Greek ideas

### DISCUSSION THREAD

**Discussion by otseng** [eliranwong/UniqueBible#762] [Ideas]
I'm trying to learn Greek.  I think it'd be nice to have tools in UBA to help people to learn Greek.  

What would you like to see in UBA to be able to learn Greek?  
Are there anything you've seen in other software that would be nice to have in UBA?

**Comment by darrelwright** [comment]
theWord has a number of premium sources at thewordbooks.com. Bill Mounce has some excellent material at https://www.billmounce.com, some free and some you purchase along with his book "Greek for the Rest of Us"
